apps/tangle-dapp/data/restake/useRestakeOperatorMap.ts

```diff
@@ -39,37 +30,35 @@ export default function useRestakeOperatorMap(): UseRestakeOperatorMapReturnType
     if (!isDefined(apiRx.query?.multiAssetDelegation?.operators?.entries))
       return of<OperatorMap>({});
 
-    return apiRx.query.multiAssetDelegation.operators
-      .entries<Option<PalletMultiAssetDelegationOperatorOperatorMetadata>>()
-      .pipe(
-        map((entries) => {
-          return entries.reduce(
-            (operatorsMap, [accountStorage, operatorMetadata]) => {
-              if (operatorMetadata.isNone) return operatorsMap;
-
-              const accountId = accountStorage.args[0];
-              const operator = operatorMetadata.unwrap();
-              const { delegations, restakersCount } = toPrimitiveDelegations(
-                operator.delegations,
-              );
-
-              const operatorMetadataPrimitive = {
-                stake: operator.stake.toBigInt(),
-                delegationCount: operator.delegationCount.toNumber(),
-                bondLessRequest: toPrimitiveRequest(operator.request),
-                delegations,
-                restakersCount,
-                status: toPrimitiveStatus(operator.status),
-              } satisfies OperatorMetadata;
-
-              return Object.assign(operatorsMap, {
-                [accountId.toString()]: operatorMetadataPrimitive,
-              } satisfies OperatorMap);
-            },
-            {} as OperatorMap,
-          );
-        }),
-      );
+    return apiRx.query.multiAssetDelegation.operators.entries().pipe(
+      map((entries) => {
+        return entries.reduce(
+          (operatorsMap, [accountStorage, operatorMetadata]) => {
+            if (operatorMetadata.isNone) return operatorsMap;
+
+            const accountId = accountStorage.args[0];
+            const operator = operatorMetadata.unwrap();
+            const { delegations, restakersCount } = toPrimitiveDelegations(
+              operator.delegations,
+            );
+
+            const operatorMetadataPrimitive = {
+              stake: operator.stake.toBigInt(),
+              delegationCount: operator.delegationCount.toNumber(),
+              bondLessRequest: toPrimitiveRequest(operator.request),
+              delegations,
+              restakersCount,
+              status: toPrimitiveStatus(operator.status),
+            } satisfies OperatorMetadata;
+
+            return Object.assign(operatorsMap, {
+              [accountId.toString()]: operatorMetadataPrimitive,
+            } satisfies OperatorMap);
```

Comment:
Curious: why is `Object.assign` being used here? Perhaps a brief comment might help for future readers